src/app/dashboard/dashboard.component.html

```diff
@@ -22,12 +23,15 @@ <h2>{{ projectBuild.project.name }}</h2>
     </div>
   </section>
 
-  <section class="columns" *ngIf="projectsBuilds.length == 0">
+  <section class="columns" *ngIf="projectsBuilds?.length == 0">
     <div class="project project-empty">
       <h3>No Projects Found.</h3>
-      <p>Have you configured Kashti and the Brigade API correctly? Check the Kashti docs for information on <a href="https://github.com/Azure/kashti/blob/angular5/docs/install.md">installing and networking</a>.</p>
-      <p>If you have yet to setup a project, see our guides for <a href="https://github.com/Azure/brigade/blob/master/docs/intro/tutorial03.md">creating projects</a> and <a href="https://github.com/Azure/brigade/blob/master/docs/topics/projects.md">managing projects</a>.</p>
+      <p>Have you configured Kashti and the Brigade API correctly? Check the Kashti docs for information on <a href="https://github.com/Azure/kashti/blob/angular5/docs/install.md">installing
+          and networking</a>.</p>
```

Comment:
oh we should update this link to use the `master` branch here too for consistency. 

can be followed up seperately though, not a blocker

Reply:
ah good call; will issue a follow-up 👍